docs/blog/posts/move-to-litellm-proxy.md

```diff
@@ -0,0 +1,25 @@
+---
+authors:
+  - Amir
+category:
+  - Architecture
+
+date: 2024-04-05
+---
+# Model calls made flexible
+
+Instead of being handled directly in our code, our model calls are now routed through an LiteLLM Proxy server, which lets use change models on the fly and have retries, fallbacks, budget tracking, and more.
```

Comment:
```suggestion
Instead of being handled directly in our code, our model calls are now routed through an LiteLLM Proxy server, which lets us change models on the fly and have retries, fallbacks, budget tracking, and more.
```